nap/engine.py

```diff
@@ -470,11 +470,18 @@ def get_from_cache(self, request_method, url):
         if cached_response:
             self.logger.debug("Got cached response for %s" % cache_key)
 
-            # Cached responses should not get re-cached to allow for
-            # expected timeouts. Now that we've retrieved the cached
-            # response, behave as if cache is turned off.
-            cached_response.use_cache = False
-            return cached_response
+            if not isinstance(cached_response, NapResponse):
+                self.logger.error("Expected to get a nap object from cache, but got %s instead: %s",
```

Comment:
instead of "nap object" you could say "NapResponse" to be most specific.

Reply:
Sure, will do.